db/re/item_db_usable.yml

```
@@ -78155,6 +78155,8 @@ Body:
       NoGuildStorage: true
       NoMail: true
       NoAuction: true
+    Script: |
+      sc_start SC_CONTENTS_1,900000,10;
```

There are specialeffect on player according to https://www.divine-pride.net/database/item/102780 (same effects on my data)
```suggestion
      sc_start SC_CONTENTS_1,900000,10;
      specialeffect2 EF_POTION_BERSERK;
      specialeffect2 EF_PROVIDENCE;
```